db/dcrpg/queries.go

```diff
@@ -1092,7 +1093,7 @@ func RetrieveBlockHeight(db *sql.DB, hash string) (height int64, err error) {
 }
 
 func RetrieveAddressTxnOutputWithTransaction(db *sql.DB, address string, currentBlockHeight int64) ([]apitypes.AddressTxnOutput, error) {
-	var outputs []apitypes.AddressTxnOutput
+	outputs := make([]apitypes.AddressTxnOutput, 0)
```

Comment:
Why the `make`?  
If you know the required capacity, you could do `make([]type, 0, cap)`, to optimize for `append`, but it's fine to append to a slice made with `var s []type`

Reply:
No problem.  I think it just a little confusion on where we want := declarations vs var x [] declarations :)